nuVeto/resources/mu/mu.py

```diff
@@ -41,6 +41,11 @@ def int_ef(preach, plight):
             preach = pickle.load(gzip.open(preach, 'rb'), encoding='latin1')
         except IOError:
             preach = hist_preach(preach)
+    elif isinstance(preach, str) and ".pklz" in preach:
```

Comment:
```suggestion
    elif Path(preach).suffix == '.pklz' in preach:
```

I think you can also get rid of the `isinstance(preach, str)` in the preceding `if` since the `Path()` function will handle it.